docs/cli/releases.mdx

```diff
@@ -133,7 +133,7 @@ Release artifacts are only considered at time of event processing. So while it
 The first argument to `upload` is the path to the file, the second is an optional URL we should associate it with. Note that if you want to use an abbreviated URL (eg: `~/foo.js`) make sure to use single quotes to avoid the expansion by the shell to your home folder.
```

Comment:
```suggestion
The first argument to `upload` is the path to the file. The second is an optional URL we should associate it with. If you want to use an abbreviated URL (`~/foo.js`, for example), make sure you're using single quotes to avoid the shell expansion to your home folder.
```

Reply:
Included in https://github.com/getsentry/sentry-docs/pull/10288